include/TGUI/Widgets/ChildWindow.hpp

```diff
@@ -199,6 +214,43 @@ namespace tgui
 
 
         /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+        /// @brief Changes the title buttons.
+        ///
+        /// @param buttons  Which buttons should be available in the title bar?
+        ///
+        /// By default ChildWindows only display a close button. You may set the window to show a combination of buttons.
+        /// For example, the following will set the ChildWindow to have both a minimize and close button.
+        /// @code
+        /// childWindow->setTitleButtons(tgui::ChildWindow::TitleButtons::Minimize | tgui::ChildWindow::TitleButtons::Close);
```

Comment:
Perhaps the "tgui::ChildWindow" bit is too verbose? I think it's implicit and can be removed without changing clarity of documentation.